source/Calamari.Testing/EnvironmentVariables.cs

```diff
@@ -70,31 +77,46 @@ public enum ExternalVariable
         [EnvironmentVariable("CALAMARI_AUTHPASSWORD", "OctopusMyGetTester")]
         MyGetFeedPassword,
 
-        [EnvironmentVariable("GOOGLECLOUD_OCTOPUSAPITESTER_JSONKEY", "GoogleCloud - OctopusAPITester")]
+        [EnvironmentVariable("GOOGLECLOUD_OCTOPUSAPITESTER_JSONKEY", "GoogleCloud - OctopusAPITester", "op://Calamari Secrets for Tests/Google Cloud Octopus Api Tester JsonKey")]
         GoogleCloudJsonKeyfile,
-        
+
         [EnvironmentVariable("GitHub_RateLimitingPersonalAccessToken", "GitHub test account PAT")]
         GitHubRateLimitingPersonalAccessToken,
     }
 
     public static class ExternalVariables
     {
+        static readonly bool SecretManagerIsEnabled = Convert.ToBoolean(Environment.GetEnvironmentVariable("CALAMARI__Tests__SecretManagerEnabled") ?? "True");
+        static readonly string SecretManagerAccount = Environment.GetEnvironmentVariable("CALAMARI__Tests__SecretManagerAccount") ?? "octopusdeploy.1password.com";
+
+        static readonly Lazy<SecretManagerClient> SecretManagerClient = new(LoadSecretManagerClient);
+
+        static SecretManagerClient LoadSecretManagerClient()
+        {
+            var allVariableAttributes = EnvironmentVariableAttribute.GetAll()
+                                                                    .Select(x => x.SecretReference)
+                                                                    .Where(x => x is not null)
```

Comment:
nit and/or an impossible suggestion if this codebase doesn't support it.
`WhereNotNull` does nullability better than `.Where(x => x is not null)` for whatever reason so it's a good habit to use it everywhere.

```suggestion
                                                                    .WhereNotNull()
```

Reply:
WhereNotNull is an extension method we created - furthermore - this code has been removed.